### ISSUE TITLE
Keycard - the keycard module is always busy

### ISSUE THREAD

**Issue by glitchminer** [status-im/status-desktop#9571]
# Bug Report

## Description

In 0.10 RC9 an error stating that the keycard module is busy is always shown when selecting any option from Keycard settings

![image](https://user-images.githubusercontent.com/50769329/219621312-6a52b7c8-c1f7-49b2-8061-a765ec8d943b.png)

The console displays the following:

```
keycard - listing readers
keycard - no smartcard reader found
keycard - keycard-error: no smartcard reader found
keycard - listing readers
keycard - no smartcard reader found
keycard - keycard-error: no smartcard reader found
INF 2023-02-17 11:15:55.383+01:00 keycard shared module is still running     topics="profile-section-profile-module" tid=1781685 file=module.nim:101
keycard - listing readers
keycard - no smartcard reader found
keycard - keycard-error: no smartcard reader found
```

## Steps to reproduce

Log in, navigate to keycard settings and select an option

#### Expected behavior

Keycard settings are available

#### Actual behavior

Error blocks the use of keycard settings

### Additional Information

- Status desktop version: 0.10 RC9
- Operating System: Mac

**Comment by saledjenic** [comment]
@glitchminer this is happening only in `0.10.0-rc9`, it's not the case in `master` branch.

I did some investigation and found out that the commit which fixes this issue was not included in that release.

@iurimatias please include commit `81a4d7093276d1e762cc3d3c88bbf58a4373a938`, it will fix this issue.

**Comment by iurimatias** [comment]
FYI this was already included in the 0.10 branch

**Comment by glitchminer** [comment]
@saledjenic , please can you have another look? Moving this to needs further analysis.

**Comment by saledjenic** [comment]
@iurimatias @glitchminer sorry, I missed the commit number, the right commit which need to be included is `91c8fe741413dc7ff4dacaa14a9b13a677590556`.

I missed it, cause in the commit message of that commit, I was referring to commit `81a4d7093276d1e762cc3d3c88bbf58a4373a938` and I just simply copied it to my previous message instead of commit `91c8fe741413dc7ff4dacaa14a9b13a677590556`. :)
